src/models/felt.rs

```diff
@@ -52,6 +52,28 @@ impl TryFrom<Felt252Wrapper> for Address {
     }
 }
 
+impl TryFrom<Felt252Wrapper> for u64 {
+    type Error = EthereumDataFormatError;
+
+    fn try_from(felt: Felt252Wrapper) -> Result<Self, Self::Error> {
+        match felt.to_be_digits() {
+            [0, 0, 0, d] => Ok(d),
+            _ => Err(EthereumDataFormatError::Primitive),
+        }
+    }
+}
+
+impl TryFrom<Felt252Wrapper> for u128 {
+    type Error = EthereumDataFormatError;
+
+    fn try_from(felt: Felt252Wrapper) -> Result<Self, Self::Error> {
+        match felt.to_be_digits() {
+            [0, 0, d1, d2] => Ok(Self::from(d1) << 64 | Self::from(d2)),
+            _ => Err(EthereumDataFormatError::Primitive),
+        }
+    }
+}
```

Comment:
Can't we use directly this

https://github.com/starknet-io/types-rs/blob/7ef30234da7957f7585ea832d57d13083c76995c/crates/starknet-types-core/src/felt/primitive_conversions.rs#L91-L96

by doing something like

```rust
let a = Felt252Wrapper;
let b: u64 = a.0.try_into().expect("");
```